data/language/en-GB.txt

```diff
@@ -2242,14 +2242,14 @@ STR_3235    :Show financial options
 STR_3236    :Show guest options
 STR_3238    :No Money
 STR_3239    :Make this park a ‘no money’ park with no financial restrictions
-STR_3240    :{WINDOW_COLOUR_2}Initial cash:
```

Comment:
Very happy with this, the label should take care of the colour code and not the string.